### ISSUE TITLE
Failed to apply harmony patches,  SE version 1.204.017 b0

### ISSUE THREAD

**Issue by tgkgate** [viktor-ferenczi/performance-improvements#60]
```
2024-05-17 17:17:08.361 - Thread:   1 ->  Info: PerformanceImprovements: Loading
2024-05-17 17:17:08.436 - Thread:   1 ->  Debug: PerformanceImprovements: Scanning for conflicting code changes
2024-05-17 17:17:08.817 - Thread:   1 ->  Debug: PerformanceImprovements: Applying Harmony patches
2024-05-17 17:17:10.673 - Thread:   1 ->  Critical: PerformanceImprovements: Failed to apply Harmony patches
[HarmonyException] Patching exception in method System.Void VRage.Groups.MyGroups`2<Sandbox.Game.Entities.MyCubeGrid, Sandbox.Game.Entities.MyGridLogicalGroupData>::MergeGroups(VRage.Groups.Group<Sandbox.Game.Entities.MyCubeGrid, Sandbox.Game.Entities.MyGridLogicalGroupData> groupA, VRage.Groups.Group<Sandbox.Game.Entities.MyCubeGrid, Sandbox.Game.Entities.MyGridLogicalGroupData> groupB)
Method: Void ReportException(System.Exception, System.Reflection.MethodBase)
Traceback:
   at HarmonyLib.PatchClassProcessor.ReportException(Exception exception, MethodBase original)
   at HarmonyLib.PatchClassProcessor.Patch()
   at HarmonyLib.CollectionExtensions.Do[T](IEnumerable`1 sequence, Action`1 action)
   at Shared.Patches.PatchHelpers.HarmonyPatchAll(IPluginLogger log, Harmony harmony)
Inner exception:

[ArgumentException] The given generic instantiation was invalid.
Method: System.Reflection.MethodInfo UpdateWrapper(System.Reflection.MethodBase, HarmonyLib.PatchInfo)
Traceback:
   at HarmonyLib.PatchFunctions.UpdateWrapper(MethodBase original, PatchInfo patchInfo)
   at HarmonyLib.PatchClassProcessor.ProcessPatchJob(Job job)
```

**Comment by viktor-ferenczi** [comment]
It is a known issue. It is expected to be fixed in the coming days once I have full access to my development tools again.

**Comment by viktor-ferenczi** [comment]
Fixed in 1.11.8 release
